docs/DeveloperGuide.md

```diff
@@ -147,6 +147,16 @@ The `Storage` component,
 * inherits from both `AddressBookStorage` and `UserPrefStorage`, which means it can be treated as either one (if only the functionality of only one is needed).
 * depends on some classes in the `Model` component (because the `Storage` component's job is to save/retrieve objects that belong to the `Model`)
 
+### Task component
```

Comment:
Task component should be within model, cause its a package in model

Reply:
Changed!